dev/bots/test.dart

```diff
@@ -457,6 +457,11 @@ Future<void> _runFrameworkTests() async {
     await _runFlutterTest(path.join(flutterRoot, 'packages', 'flutter_localizations'), tableData: bigqueryApi?.tabledata);
     await _runFlutterTest(path.join(flutterRoot, 'packages', 'flutter_test'), tableData: bigqueryApi?.tabledata);
     await _runFlutterTest(path.join(flutterRoot, 'packages', 'fuchsia_remote_debug_protocol'), tableData: bigqueryApi?.tabledata);
+    await _runFlutterTest(
+      path.join(flutterRoot, 'packages', 'flutter', 'tracing'),
```

Comment:
@yjbanov - do you have any complaints about putting this in package/flutter?

This depends on dart:io and the io methods of connecting to the vm service.  I'm doing this to avoid writing a driver test, since we're not actually using this for performance just yet.  I have it marked as skip if browser.

Reply:
I've thought the better of this.  Having this test under packages/flutter means a regular person can't just run `flutter test` in that folder without at least one test failing.  Moved this to `dev/tracing` with a README.